spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -63,6 +64,21 @@ def get_unique_figname(dirname, root, ext):
             return osp.join(dirname, figname)
 
 
+@Slot(object, str)
+def copy_figure(fig, fmt):
```

Comment:
Could you use more descriptive variable names here?

Reply:
Unless its to be consistent with other (poor) names nearby...

Reply:
There are these variables names(fig, fmt) in figurebrowser.py. If I modify them, there will be multiple variable names with same meaning.

Reply:
Okay, gotcha. Thanks!